easybuild/easyconfigs/n/NCO/NCO-4.7.9-foss-2018b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'NCO'
+version = '4.7.9'
+
+homepage = "http://nco.sourceforge.net"
+description = """manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/nco/nco/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['048f6298bceb40913c3ae433f875dea1e9129b1c86019128e7271d08f274a879']
+
+builddependencies = [
+    ('Bison', '3.0.5'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('UDUNITS', '2.2.26'),
+    ('expat', '2.2.5'),
+    ('ANTLR', '2.7.7', '-Python-3.6.6'),
```

Comment:
@smoors Do we really need a Python-enabled ANTLR here?

Because if we pull in Python as a runtime dep here, the `-Python-3.6.6` should trickle up into the `versionsuffix` of `NCO`...